xml/libvirt_guest_installation.xml

```diff
@@ -302,15 +302,19 @@
         </step>
         <step>
           <para>
-            Under <guimenu>Enable boot menu</guimenu>, activate
-            <guimenu>Network PXE</guimenu> and confirm with
+            Under <guimenu>Enable boot menu</guimenu> and confirm with
```

Comment:
It seems that you have deleted the text: "_, activate <guimenu>Network PXE</guimenu>_". I am very new to this guide as a maintainer, so I am not sure if the omission is intentional or accidental. But something seems off.

Reply:
There is no more this option in virt-manager